java/src/jmri/jmrix/can/cbus/swing/hubpane/CbusHubAction.java

```diff
@@ -8,7 +8,21 @@ public CbusHubAction() {
         super("CBUS Hub Control",
             new jmri.util.swing.sdi.JmriJFrameInterface(),
             CbusHubPane.class.getName(),
-            jmri.InstanceManager.getDefault(CanSystemConnectionMemo.class));
+            memo);
     }
 
+    // we have to locate the (only) CBUS CanSystemConnectionMemo
+    // before we can call the super constructors.
+    private static CanSystemConnectionMemo memo;
+    
+    static {        
+        var memos = jmri.InstanceManager.getList(CanSystemConnectionMemo.class);
+        for (CanSystemConnectionMemo check : memos) {
+            String name = check.getUserName();
+            if (name.equals("MERG")) {
```

Comment:
Will this work? If I understand the code correctly, the username can be set by the user.